debug.log

```diff
@@ -0,0 +1,9 @@
+[0129/151620.937:ERROR:registration_protocol_win.cc(130)] TransactNamedPipe: The pipe has been ended. (0x6D)
```

Comment:
What is this file? SHould we be tracking this?

Reply:
no.  I'll remove.  It popped up when running npm test